services/core-web/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js

```diff
@@ -32,39 +32,26 @@ const propTypes = {
 };
 
 const defaultProps = {
-  sortField: null,
-  sortDir: null,
+  sortField: undefined,
```

Comment:
I get your thought process here with null vs undefined, but if we're going to be nuanced in the difference between null and undefined I might argue that null is more correct here as we are setting a "default _value_" of nothing vs an unassigned variable.

Reply:
I changed them because I thought using undefined fixed some undesired behaviour. Will double check.